auraed/src/lib.rs

```diff
@@ -37,6 +38,58 @@
 //! The Aurae Daemon (auraed) is the main server implementation of the Aurae
 //! Standard Library.
 //!
+//! The Aurae Daemon runs as a gRPC server which listens over a unix domain socket by default.
+//!
+//! ```bash
+//! /var/run/aurae/aurae.sock
+//! ```
+//!
+//! ## Running Auraed
+//!
+//! Running as `/init` is currently under active development.
+//!
+//! To run auraed as a standard library server you can run the daemon alongside your current init system.
+//!
+//! ```bash
+//! sudo -E auraed
+//! ```
+//!
+//! Additional flags are listed below.
+//!
+//! ```bash
+//! USAGE:
+//!     auraed [OPTIONS]
+//!
+//! OPTIONS:
+//!         --ca-crt <CA_CRT>            [default: /etc/aurae/pki/ca.crt]
+//!     -h, --help                       Print help information
+//!     -s, --socket <SOCKET>            [default: /var/run/aurae/aurae.sock]
+//!     -r, --runtime_dir <RUNTIME_DIR>  [default: /var/run/aurae]
+//!         --server-crt <SERVER_CRT>    [default: /etc/aurae/pki/_signed.server.crt]
+//!         --server-key <SERVER_KEY>    [default: /etc/aurae/pki/server.key]
+//!     -v, --verbose
+//!     -V, --version                    Print version information
+//!
+//! ```
+//!
+//! ## Building from source
+//!
+//! We suggest using the [aurae](https://github.com/aurae-runtime/aurae) repository for building all parts of the project.
+//!
+//! It is possible to build `auraed` by itself. Check out this repository and use the Makefile.
+//!
+//! ```bash
+//! make auraed
+//! ```
+//!
+//! or using Cargo directly
```

Comment:
For now using Cargo manually is unsupported. Please remove it from the docs such that `make` is the only supported way of building Aurae.